### PR TITLE
- Lint README

### PR DESCRIPTION
**Pull request by brettz9**

### CODE REVIEW

**Comment by brettz9** [comment]
Btw, if you ever add `@example` tags within the code, ash-nazg can lint that too (just as this PR adds for Markdown files JavaScript blocks)...

File: README.md [hunk 1]
```diff
@@ -327,8 +327,8 @@ The example below shows how you might convert and intersect two arbitrary SVG el
 ```javascript
 const {SvgShapes, Intersection} = require("kld-intersections");
 
-const shape1 = SvgShapes.element(document.getElementById("a"));
-const shape2 = SvgShapes.element(document.getElementById("b"));
+const shape1 = SvgShapes.element(document.querySelector("a"));
```
Comment (thelonious) [inline README.md]: I believe this changes the semantics. My version grabs an element by id. Your version looks for the first "a" element in the document.

Comment (brettz9) [reply README.md]: Unfortunately, there is no all-purpose linter yet for absent-mindedness. Fixed (and rebased)! :)

**Comment by thelonious** [comment]
Thanks for the fixes. BTW, what's the rational for `querySelector` vs `getElementById`?

**Comment by brettz9** [comment]
I have seen one prominent dev suggest `getElementById` could even be deprecated in the future, though for backward-compatibility I'd sure hope not... But it is a good practice I think as one can more consistently search one's code if one follows one form...

**Pull request merged**